src/themes/creator.js

```diff
@@ -285,6 +285,21 @@ export default function createTheme(
     tooltip: {
       backgroundColor: primitives.mono900,
     },
+    grid: {
```

Comment:
these are not yet final values for base scale. Will be changed upon design team sharing with us values for base scales, spacing, etc. Also LightTheme is not final place to put these constants. If we make it as part of themes, we will be able to customize it as part of theme, but need to repeat it in all of our themes.